docs/modules/ROOT/pages/reference/extensions/quarkus-cxf-integration-tracing-opentelemetry.adoc

```diff
@@ -15,10 +15,12 @@ endif::[]
 
 Generate https://opentelemetry.io/[OpenTelemetry] https://opentelemetry.io/docs/concepts/signals/traces/[traces].
 
-IMPORTANT: OpenTelemetry Metrics and Logging are not supported yet on neither Quarkus nor CXF side,
-           hence {quarkus-cxf-project-name} cannot support them either. Tracing is thus
-           the only OpenTelemetry feature supported by this extension.
-
+[IMPORTANT]
+====
+OpenTelemetry Metrics and Logging are not supported yet on neither Quarkus nor CXF side,
+hence {quarkus-cxf-project-name} cannot support them either. Tracing is thus
+the only OpenTelemetry feature supported by this extension.
+====
```

Comment:
This file is generated. You need to change the source file `extensions/integration-tracing-opentelemetry/runtime/src/main/doc/intro.adoc`. Otherwise your change gets overwritten by the plugin generating this page. You may want to run `./mvnw clean install -DskipTests -Dquarkus.build.skip` from the root dir to see what gets overridden and where you need to find the source file. 
Pro tip: use `mvnd` instead of `mvnw`/`mvn`